README.md

```diff
@@ -106,13 +106,13 @@ Wikibase Internal Serialization has been written by [Jeroen De Dauw]
 
 ## Release notes
 
-### 1.4 (dev)
+### 1.4 (2015-06-12)
 
-* Added PHPMD and PHPCS support
-* Added support for showing the component version when loaded via MediaWiki
-* Added compatibility with DataModel 3.x and DataModelSerialization 1.4
+* Added compatibility with DataModel 3.x
```

Comment:
It's not just "added compatibility", it actually requires DataModel 3 now.

Personally I think it can't hurt to mention the other updates (DataModel Serialization 1.4, possibly Geo and Time), but they are not crucial.